client/packages/system/src/Stock/ListView/ListView.tsx

```diff
@@ -111,34 +111,34 @@ export const StockListView: FC = () => {
       {
         id: 'vvmStatus',
         header: t('label.vvm-status'),
-        size: 150,
         accessorKey: 'vvmStatus?.description',
```

Comment:
For ones like this that can return `undefined`, you get a warning like this: 

<img width="406" height="161" alt="Image" src="https://github.com/user-attachments/assets/d56e0c08-259a-4c0f-b41d-4c405c0483df" />

So best to just convert to an accessorFn with a fallback:
```ts
accessorFn: row => row.vvmStatus?.description ?? '',
```

Similarly for `location.code` below...

Reply:
Sorry did this with a search and replace I think, didn't check properly good catch